packages/app/components/stackbox/Stackbox.tsx

```diff
@@ -70,6 +86,14 @@ export const Stackbox = () => {
     setEndDateTime(new Date(endDateByFrequency[frequency]));
   }, [frequency]);
 
+  const formattedBalance = (balanceData: NonNullable<typeof balance>) =>
```

Comment:
this function could be on `utils/token` file

Reply:
I think we should uniformize all this formates in the app. And add this formatting to a specific file. If you agree will create a task to do this. But we can keep this function inside Stackbox for this PR.

Reply:
Agree on standardizing formats throughout the app but I agree with Diogo to abstract this function into a `utils` file, because, in my experience, these functions tend to grow and change their signature as the app needs to grow. At some point, I'm sure we will abstract this function, sooner or later.

I don't mind handling that abstraction in a different PR as long as we handle it, either here or in another task.